test/e2e/network/hostport.go

```diff
@@ -143,11 +143,19 @@ var _ = common.SIGDescribe("HostPort", func() {
 			}
 			// check pod3
 			ginkgo.By(fmt.Sprintf("checking connectivity from pod %s to serverIP: %s, port: %d UDP", hostExecPod.Name, hostIP, port))
-			_, _, err = f.ExecCommandInContainerWithFullOutput(hostExecPod.Name, "e2e-host-exec", cmdPod3...)
+			hostname3, _, err := f.ExecCommandInContainerWithFullOutput(hostExecPod.Name, "e2e-host-exec", cmdPod3...)
 			if err != nil {
 				framework.Logf("Can not connect from %s to pod(pod2) to serverIP: %s, port: %d", hostExecPod.Name, hostIP, port)
 				continue
 			}
+			if hostname1 == hostname3 {
+				framework.Logf("pods must have different hostname: pod1 has hostname %s, pod3 has hostname %s", hostname1, hostname3)
+					continue
+			}
+			if hostname2 == hostname3 {
+				framework.Logf("pods must have different hostname: pod2 has hostname %s, pod3 has hostname %s", hostname2, hostname3)
+					continue
```

Comment:
ditto

Reply:
Yeah, I wonder how this happened, because I did test this locally. Hmm.

I've pushed a new version

Reply:
Ok, I think I know what happened. Before pushing I did press `=` in visual mode in vim. This is, probably, what I do subconsciously when writing in C :rofl: Need to get used to Go

Reply:
:smile: